Sources/ArcGISToolkit/Components/Augmented Reality/CalibrationView.swift

```diff
@@ -58,6 +62,7 @@ extension WorldScaleGeoTrackingSceneView {
                                 .font(.body.smallCaps())
                                 .foregroundStyle(.secondary)
                             Spacer()
+                            Text(heading.isLess(than: 0) || heading.isZero ? "" : "+") +
```

Comment:
you can send up with a "+0°" in the text because it's not comparing the rounded number we show in the text to zero, but the actual value.